node/service/src/service.rs

```diff
@@ -63,60 +66,32 @@ pub mod frequency_executor {
 
 pub use frequency_executor::*;
 
+type ParachainExecutor = NativeElseWasmExecutor<FrequencyRuntimeExecutor>;
```

Comment:
Note for future readers of this PR. Most of these edits are following https://github.com/substrate-developer-hub/substrate-node-template/ so that future upgrades are easier to follow.

Reply:
Also https://github.com/paritytech/cumulus/parachain-template/node, correct?